src/styles.scss

```diff
@@ -1868,3 +1898,55 @@ input {
     opacity: 1;
   }
 }
+
+// Added BootStrap and Custom CSS for Themeing
+.border-bottom {
+  border-bottom: 1px solid var(--secondary) !important;
+}
+
+.border-top {
+  border-top: 1px solid var(--secondary) !important;
+}
+
+.border-left {
+  border-left: 1px solid var(--secondary) !important;
+}
+
+.border-right {
+  border-right: 1px solid var(--secondary) !important;
+}
+
+.text-black {
+  color: var(--text) !important;
+}
+
+.table {
+  color: var(--text) !important;
+}
+
+.clout {
+  filter: var(--filter);
```

Comment:
This filter causes a colour change on the bitclout logo. Shows kinda purple when in light mode. All good in dark mode.

Reply:
Yea oops need to fix that, we can just not have it change anything on light theme selection. Commit incoming.